typescript/web/src/components/text-link.tsx

```diff
@@ -0,0 +1,14 @@
+import { Text } from "@chakra-ui/react";
+import NextLink from "next/link";
+import { PropsWithChildren } from "react";
+
+export type TextLinkProps = PropsWithChildren<{
+  href: string;
+}>;
+export const TextLink = ({ children, href }: TextLinkProps) => (
```

Comment:
```suggestion
}>;

export const TextLink = ({ children, href }: TextLinkProps) => (
```

Have you added the following lines in your own VS Code settings.json as suggested earlier?

```json
  "codemetrics.basics.CodeLensEnabled": false,
```

Bonus, the line below makes the score more realistic with early returns:
```json
  "codemetrics.nodeconfiguration.ReturnStatement": 0,
```